contracts/Datastore.sol

```diff
@@ -334,10 +397,9 @@ contract Datastore is AragonApp {
      * @notice Add a group to the datastore
      * @param _groupName Name of the group
      */
-    function createGroup(string _groupName) external auth(DATASTORE_MANAGER_ROLE) returns (uint) {
-        uint id = groups.createGroup(_groupName);
+    function createGroup(string _groupName) external {
+        groups.createGroup(_groupName);
         emit GroupChange(msg.sender);
```

Comment:
Maybe a more specific event for group creation would be better. Also, including the group id?

Reply:
Yes now that the cleanup is done for the events, we are considering adding a NewGroup event or maybe add a parameter to GroupChange that will specify the type of change